Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -63,7 +54,8 @@ class PostRideSummaryViewController: PostRideViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
-        guard let creator = currentRide?.creator else { return }
+        self.currentRide = NetworkManager.shared.currentRide
+        
         setUpStackView()
         setUpButton()
```

Comment:
Okay, nvm this is my bad. I think my brain just spontaneously decided to capitalize the "U" when I made this controller lol. Do you mind changing them to "setup" for me 👉👈

Reply:
ofc richie :>